src/Temporalio/Bridge/OptionsExtensions.cs

```diff
@@ -483,6 +494,86 @@ internal static class OptionsExtensions
             };
         }
 
+        private static Interop.TunerHolder ToInteropTuner(
+            this Temporalio.Worker.Tuning.WorkerTuner tuner,
+            Scope scope)
+        {
+            Temporalio.Worker.Tuning.ResourceBasedTunerOptions? lastTunerOptions = null;
+            Temporalio.Worker.Tuning.ISlotSupplier[] suppliers =
+            {
+                tuner.GetWorkflowTaskSlotSupplier(), tuner.GetActivityTaskSlotSupplier(),
+                tuner.GetLocalActivitySlotSupplier(),
+            };
+            foreach (var supplier in suppliers)
+            {
+                if (supplier is Temporalio.Worker.Tuning.ResourceBasedSlotSupplier resourceBased)
+                {
+                    if (lastTunerOptions != null && lastTunerOptions != resourceBased.TunerOptions)
+                    {
+                        throw new ArgumentException(
+                            "All resource-based slot suppliers must have the same ResourceBasedTunerOptions");
+                    }
+
+                    lastTunerOptions = resourceBased.TunerOptions;
+                }
+            }
+
+            return new()
+            {
+                workflow_slot_supplier =
+                    tuner.GetWorkflowTaskSlotSupplier().ToInteropSlotSupplier(true),
+                activity_slot_supplier =
+                    tuner.GetActivityTaskSlotSupplier().ToInteropSlotSupplier(false),
+                local_activity_slot_supplier =
+                    tuner.GetLocalActivitySlotSupplier().ToInteropSlotSupplier(false),
+            };
+        }
+
+        private static Interop.SlotSupplier ToInteropSlotSupplier(
+                this Temporalio.Worker.Tuning.ISlotSupplier supplier,
+                bool isWorkflow)
```

Comment:
```suggestion
            this Temporalio.Worker.Tuning.ISlotSupplier supplier,
            bool isWorkflow)
```

Inconsistent spacing

Reply:
Dotnet format seems to just... not format stuff I would expect it to fairly often

Reply:
Yeah, I don't use it to rewrite code, I just use it to know CI will pass